erpnext/erpnext_integrations/taxjar_integration.py

```diff
@@ -57,7 +64,10 @@ def create_transaction(doc, method):
 	tax_dict['amount'] = doc.total + tax_dict['shipping']
 
 	try:
-		client.create_order(tax_dict)
+		if tax_dict['amount']<0:
```

Comment:
```suggestion
		if doc.is_return:
```

There could be cases where a regular Sales Invoice might have a negative amount tax row, better to check if Invoice is a credit note